lib/brook/server.ex

```diff
@@ -54,13 +74,13 @@ defmodule Brook.Server do
           nil
       end
     end)
-
-    {:reply, :ok, state}
   end
 
-  def handle_call({:send, type, event}, _from, state) do
-    :ok = apply(state.driver.module, :send_event, [type, event])
-    {:reply, :ok, state}
+  defp process(%{forwarded: true} = event, state) do
+    IO.inspect(event, label: "Forwarded Event")
```

Comment:
Same here on the `IO.inspect` -> `Logger.debug/info`

Reply:
fixed